source/privacy-notice.html.erb

```diff
@@ -10,166 +10,145 @@ description: Find details about the GovWifi privacy policy and how we collect an
-        <p class="govuk-body">
-          The data controller for your personal data is the Cabinet Office.
-        </p>
-        <p class="govuk-body">Contact the Data Protection Officer if you either:</p>
-        <ul class="govuk-list govuk-list--bullet">
-          <li>have any questions about anything in this document</li>
-          <li>think that your personal data has been misused or mishandled</li>
-        </ul>
-        <h3 class="govuk-heading-s">Data Protection Officer</h3>
-        <p class="govuk-body">
-          Cabinet Office
-          <br/>
-          70 Whitehall
-          <br/>
-          London
-          <br/>
-          SW1A 2AS
-        </p>
+<p class="govuk-body">We require this information to ensure that our research is varied and representative of our user groups.</p> 
+
+<p class="govuk-body">This data gived us insights from end users and organisations to help us identify ways we can improve GovWifi. It allows us to capture pain points and to broaden our knowledge of how the service is used and seen.</p> 
+
+
+        <h2 class="govuk-heading-m" id="legal-basis">Legal basis for processing your data</h2>
+
         <p class="govuk-body">
-          Email
-          <br/>
-          <%= link_to "DPO@cabinetoffice.gov.uk", "mailto:DPO@cabinetoffice.gov.uk", class: "govuk-link" %>
+          The legal basis for processing your personal data is:
         </p>
-        <p class="govuk-body">
-          If you have a complaint, you can also contact the <%= link_to "Information Commissioner", "https://ico.org.uk", class: "govuk-link" %>, who is an independent regulator set up to uphold information rights.
+    
+       <p class="govuk-body">
+Processing is necessary for the performance of a task carried out in the public interest or in the exercise of official authority vested in the data controller. In this case that is the running of wifi services.
+          </p>
+  <h2 class="govuk-heading-m" id="user-research">For user research</h2>
+     <p class="govuk-body">The initial use of personal data to send surveys (when data subjects sign up for GovWifi credentials) is on a lawful basis of legitimate interest.</p>
+
+<p class="govuk-body">All other processing for user research is on a lawful basis of consent.</p>
+
+   <h2 class="govuk-heading-m" id="recipients">Recipients</h2>
+ 
+<p class="govuk-body">Your personal data may be shared with other Government department in specific circumstances such as investigations: </p>
+ <ul class="govuk-list govuk-list--bullet">
+          <li>Legal investigations</li>
+  <li>Cyber Security investigations</li>
+  <li>HR employment investigations</li>
+  <li>If required by court order</li></ul>
+
+<p class="govuk-body">We will not:</p>
+<ul class="govuk-list govuk-list--bullet">
+          <li>sell or rent your data to third parties</li>
+<li>share your data with third parties for marketing purposes</li>
+<li>We do not share data that monitor or track employees</li></ul>
```

Comment:
Don't need the "We do not"
change monitor into monitors
change track into tracks

Reply:
can you re-approve the PR, ta.